azure-pipelines/test.yml

```diff
@@ -89,9 +89,10 @@ jobs:
   - script: mv *.tgz package.tgz
     displayName: 'Rename v4 tgz file' # because the publish command below requires an exact path
     workingDirectory: '$(Pipeline.Workspace)/nodeLibraryV4/drop'
-  - script: npm install "$(Pipeline.Workspace)/nodeLibraryV4/drop/package.tgz"
-    displayName: 'install latest v4 library'
-    workingDirectory: 'app/v4'
+  # TODO: Temporary change until actual v4 drop exists with http stream changes
```

Comment:
Will revert these changes and remove "app/v4/azure-functions-4.1.0.tgz" after stream support is merged in the library package and before merging this PR